packages/react-server/src/ReactFizzComponentStack.js

```diff
@@ -63,6 +72,9 @@ export function getStackByComponentStackNode(
         case 2:
           info += describeClassComponentFrame(node.type);
           break;
+        case 3:
```

Comment:
Do we encounter these in prod considering these are dev-only?

Reply:
No. We can gate it.